sql/upgrade/1.4.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -355,8 +355,68 @@ CREATE TABLE `t_ds_environment` (
    UNIQUE KEY `environment_code_unique` (`code`)
 ) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;
 
-ALTER TABLE t_ds_task_definition ADD COLUMN `environment_code` bigint(20) default '-1' COMMENT 'environment code' AFTER `worker_group`;
-ALTER TABLE t_ds_task_definition_log ADD COLUMN `environment_code` bigint(20) default '-1' COMMENT 'environment code' AFTER `worker_group`;
```

Comment:
Why remove this part of sql?

Reply:
Because it already exists in the added t_ds_task_definition, t_ds_task_definition_log table creation statement.

Reply:
sorry, i didn't notice.